dll/win32/shell32/CDefView.cpp

```diff
@@ -2225,6 +2225,11 @@ LRESULT CDefView::OnNotify(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &bHandl
                     PIDLIST_ABSOLUTE pidlFull = ILCombine(m_pidlParent, pidl);
                     SHGetPathFromIDListW(pidlFull, szFullPath);
 
+                    INT cchLimit = 0;
+                    _DoFolderViewCB(SFVM_GETNAMELENGTH, (WPARAM)pidlFull, (LPARAM)&cchLimit);
+                    if (cchLimit)
+                        ::PostMessageW(hEdit, EM_SETLIMITTEXT, cchLimit, 0);
```

Comment:
Why using PostMessage instead of SendMessage (here and probably the two other calls below), in order to ensure the edit control correctly updates itself before the rest of the code can run (if it ever matters)?

Reply:
Used `SendMessageW` instead.